docs/_articles/en/codebuilder/cb-uninstall.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Uninstall Code Builder
+lang: en
+---
+
+## Uninstall Code Builder
+
+You can uninstall Code Builder, if necessary. If you plan to reinstall Code Builder, don't delete the Code Builder auth providers. However, if you deleted the auth providers, you can recreate them by re-enabling the Code Builder preference in Setup.
+
+**Important**: Uninstalling Code Builder also removes all associated data including, but not limited to, projects, and activity history.
+
+### Remove Permission Set Assignments
+
+Before you uninstall the Code Builder package, remove permission set assignments, then delete Code Builder permissions sets.
+
+First, remove the Code Builder permission set group assignments from all Code Builder users.
+
+- CodeBuilderGroup
+
+Next, delete the Code Builder permission sets.
+
+From Setup, enter `Permission Sets` in the Quick Find box, then select **Permission Sets**.
+Delete the `Code Builder Package` permission set.
+
+### Uninstall the Code Builder Package
+
+After you remove the permission set and permission set assignments, you can uninstall the Code Builder package, which removes the application and all its associated data and objects.
+
+Before you uninstall the package:
+
+- Remove Code Builder permission Set assignments.
+- Delete Code Builder permission sets.
+
+1. From Setup, enter `Installed Packages` in the Quick Find box, then select **Installed Packages**.
+2. Click **Uninstall**, then scroll to the bottom of the page to choose whether to save and export a copy of the package’s data.
+3. Select **Yes, I want to uninstall this package and permanently delete all associated components**.
+4. Click **Uninstall**.
+
+### Disable Code Builder Preference
```

Comment:
@sbudhirajadoc - FYI - If you disable this, it also blocks user access to Code Builder, even if they still have the permission set group assigned.  

Do users know what about the "preference" under the hood.  Or should we just title this section "Disable Code Builder"?